### DISCUSSION TITLE
Feature ideas

### DISCUSSION THREAD

**Discussion by SocialForge** [SocialForge/InstaSweep#2] [Ideas]
General discussion for brainstorming future features and for keeping track of them.

Suggestions are welcome from everybody.

---

**Disclaimer:** It's important to mention that not necessarily all the suggestions will be implemented.
This is merely a list to provide target ideas for future implementation.
In reality, whether or not to implement a suggestion is based on it's complexity, user demand, whether or not it makes sense, and general feasibility of the idea.

---

### Suggestions

- [ ] Add "new" badge for unfollowers that weren't there the last time the user used the tool
- [ ] Allow setting sleep time for scanning and unfollowing
- [ ] Limit unfollows to max amount allowed by IG
- [ ] Allow saving unfollowers to unfollow later
- [ ] Allow filtering based on account followers
- [ ] Provide a way to start/stop the tool when running (during scanning or unfollowing or both)
- [ ] Add option to block selected unfollowers

**Comment by aracyla** [comment]
Don't you think that this would be more useful as browser plugin?

**Reply by SocialForge** [reply]
Yeah I've been thinking about it for some time.
Ideally I would like users to have both options available, the copy-paste option and the extension option.
And perhaps even some other methods in the future.

So it is in the works :)
